### PR TITLE
Added computed whole range property to String.

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

**Review by yuliaveres — APPROVED** [review]

File: Sources/String+Sweetness.swift [hunk 1]
```diff
@@ -37,6 +37,10 @@ public extension NSRange {
 }
 
 public extension String {
+    public var wholeRange: Range<String.Index> {
+        return self.range(of: self)!
```
Comment (yuliaveres) [inline Sources/String+Sweetness.swift]: what if the string is empty ?

Comment (yuliaveres) [reply Sources/String+Sweetness.swift]: should we force unwrap ? ..

**Review by yuliaveres — COMMENTED** [review]
🤔

**Review by yuliaveres — CHANGES_REQUESTED** [review]
🤔

File: Sources/String+Sweetness.swift (hunk 1 above)
Comment (designatednerd) [reply Sources/String+Sweetness.swift]: Definitely worth adding a test case on empty string

Comment (elland) [reply Sources/String+Sweetness.swift]: Added test. Was surprised to learn that it does crash.

👍

**Review by yuliaveres — APPROVED** [review]
